pkg/providers/aws/tags.go

```diff
@@ -23,6 +30,26 @@ func ec2TagsToGeneric(ec2Tags []*ec2.Tag) []*tag {
 	return genericTags
 }
 
+func genericToEc2Tag(tag *tag) *ec2.Tag {
+	var ec2Tag *ec2.Tag
+	ec2Tag = &ec2.Tag{Key: aws.String(tag.key), Value: aws.String(tag.value)}
+	return ec2Tag
+}
+
+func genericToRdsTag(tag *tag) *rds.Tag {
+	var rdsTag *rds.Tag
+	rdsTag = &rds.Tag{Key: aws.String(tag.key), Value: aws.String(tag.value)}
+	return rdsTag
+}
```

Comment:
That's pretty cool, doing it both ways 👍  We can return them inline if we want to, like

```suggestion
func genericToEc2Tag(tag *tag) *ec2.Tag {
	return &ec2.Tag{Key: aws.String(tag.key), Value: aws.String(tag.value)}
}

func genericToRdsTag(tag *tag) *rds.Tag {
	return &rds.Tag{Key: aws.String(tag.key), Value: aws.String(tag.value)}
}
```